spring-cloud-dataflow-module-deployers/spring-cloud-dataflow-module-deployer-kubernetes/README.md

```diff
@@ -0,0 +1,176 @@
+SPI implementation for deploying Spring Cloud Stream modules to [Kubernetes](http://http://kubernetes.io/).
+
+### Setup a Kubernetes Cluster 
+
+If you don't have access to a Kubernetes Cluster, you can deploy one locally.
+This requires a local functional Docker installation and the usage of some 
+nice scripts to setup [Kubernetes](https://github.com/olmoser/docker-compose-kubernetes).
+
+In a nutshell, do the following steps:
+```
+$ git clone https://github.com/olmoser/docker-compose-kubernetes
+$ cd docker-compose-kubernetes
+$ docker-machine create --driver virtualbox sc-dataflow-dev 
+$ docker-machine start sc-dataflow-dev 
+$ eval "$(docker-machine env sc-dataflow-dev)"
+$ ./kube-up.sh
+```
+
+Once the cluster is up, configure you local system to your local DNS server `10.0.0.10` 
```

Comment:
I don't understand this sentence. There is a typo for sure 'configure *you* local system', but even if it read "your", not sure what to do